DESCRIPTION

```diff
@@ -21,21 +21,25 @@ Depends:
 Imports: 
     dplyr (>= 1.0.0),
     ospsuite.utils (>= 1.2.0.9000),
```

Comment:
please change to `ospsuite.utils (>= 1.3)`

Reply:
That version doesn't exist.

https://github.com/Open-Systems-Pharmacology/OSPSuite.RUtils/blob/c784c6f1bee7bbd9b5a5f049ad8f2ce255fe767c/DESCRIPTION#L4

It was supposed to happen in [this](https://github.com/Open-Systems-Pharmacology/OSPSuite.RUtils/pull/98/files) PR, but it has been pending for over a week now.